src/Core/Domain/Order/Payment/Command/AddPaymentCommand.php

```diff
@@ -102,7 +102,6 @@ public function __construct(
         ?string $transactionId = null
     ) {
         $amount = new DecimalNumber($paymentAmount);
-        $this->assertAmountIsPositive($amount);
```

Comment:
Even if you add a constraint directly on the database definition, I think it's better to let this check at this point. Because with this check, the error is thrown before we try to save and we can handle the exception to prevent the user as soon as possible

Reply:
you mean it's good that you can't enter a negative value?

Reply:
Sorry, I understood the opposite... I resolve my review, waiting for PM's return.

Reply:
What is PM Return? What am I supposed to do now?